pkgs/top-level/python-packages.nix

```diff
@@ -20710,6 +20710,36 @@ in {
 
   transaction = callPackage ../development/python-modules/transaction { };
 
+  translate-toolkit = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.